dashboard/getdata.py

```diff
@@ -181,6 +221,14 @@ def main():
             csv.write(
                 "gcc_hash,hash_timestamp,libc-libname-tool,libc,target,tool,unique_fails,total_fails\n"
             )
+        with open("filtered_linux.csv", "a") as csv:
```

Comment:
- Ditto for the simplicity here. Since they all write the same, can we simplify this one with a for loop also? Then the readers know that they all write the same content easily. 
- If the download_logs don't create any of these files, `with open("filtered_linux.csv", "w")` might be better to show that this file should be newly created instead of appended.